src/SiteMaster/Core/Auditor/Site/Pages/Queued.php

```diff
@@ -35,6 +35,6 @@ public function getLimit()
 
     public function getOrderBy()
     {
-        return 'ORDER BY priority ASC, start_time ASC';
+        return 'ORDER BY priority ASC, date_created ASC';
```

Comment:
Yeah.  I'm not sure why this was built this way.  The `start_time` will always be `NULL` until the page scan starts.  In other words, everything that is queued will have a `NULL` `start_time`.  It should be ordering on the `date_created` instead.

Once a scan starts to be crawled, pages that are found will have a lower priority value, and thus the site will be scanned before a new site is crawled.